### ISSUE TITLE
Edgeyolo tflite quantization

### ISSUE THREAD

**Issue by ramonhollands** [PINTO0309/onnx2tf#387]
### Issue Type

Others

### onnx2tf version number

1.13.10

### onnx version number

Version: 1.13.1

### onnxruntime version number

Version: 1.15.0

### onnxsim (onnx_simplifier) version number

Version: 0.4.17

### tensorflow version number

Version: 2.13.0rc0

### Download URL for ONNX

https://www.dropbox.com/s/kzsk9d5kmnyebu3/all_files.zip?dl=0

### Parameter Replacement JSON

```json
I did not use this parameter replacement.
```

### Description

1. Ecologic Research. Core project need
2.

I am trying to export an onnx model generated by https://github.com/LSH9832/edgeyolo/ to tflite.

The float32 and float16 variants are fine, the model is valid and is working correctly:
Using command: onnx2tf -i edgeyolo_tiny_lrelu_coco.onnx  -ois input:1,3,640,640 -cotof -cotoa 1e-1

I used this command to export with INT8 Quantization, Full INT8 Quantization
onnx2tf -i edgeyolo_tiny_lrelu_coco.onnx -oiqt

I tried to use the parameter '--disable_group_convolution' because I saw this warning

Then I uses your script to validate the models:
https://github.com/PINTO0309/onnx2tf/issues/244

The model is corrupt at some point. 
I tried to use your suggestion to debug the model and determine the exact point like in issues/244 but I did not succeed because of strange output names shown in netron, like 'name: model/tf.math.add_12/Add;model/tf.nn.convolution_215/convolution;model/tf.nn.convolution_10/convolution;Const_324'

How would you approach this? Still debugging the model? If so where can I find the correct 'split' names?

Thanks in advance for your reaction and help!

**Comment by ramonhollands** [comment]
The onnx model is exported using the edgeyolo command:
python /edgeyolo/export.py --onnx-only --weights edgeyolo_tiny_lrelu_coco.pth --input-size 640 640 --batch 1 --opset 11

**Comment by PINTO0309** [comment]
The OP output name of tflite is a string randomly generated by TensorFlow on its own, so this tool does not use it at all. I'm not entirely sure I understand what you want to implement, but if you want to break up the model and see where the accuracy degradation is occurring, you need to specify **`the name of the ONNX output`**.

![image](https://github.com/PINTO0309/onnx2tf/assets/33194443/0915a3e6-8702-467b-8690-a5b8eb12086d)

![image](https://github.com/PINTO0309/onnx2tf/assets/33194443/50acf1c3-a842-427b-b7bd-9fc16b21ec7d)

```bash
onnx2tf -i edgeyolo_tiny_lrelu_coco.onnx -onimc /model.77/Concat_output_0 -oiqt
```

**Comment by PINTO0309** [comment]
Incidentally, this part of the model is the anti-pattern of quantization.
![image](https://github.com/PINTO0309/onnx2tf/assets/33194443/4d2b0f83-5f04-441f-b14d-49b74f4fb4d5)

**Comment by ramonhollands** [comment]
Thanks for you reply!

Why is it exactly an anti pattern?

And is this the reason the 'Dequantize' parts are showing up everywhere in the tflite graph?

![afbeelding](https://github.com/PINTO0309/onnx2tf/assets/20172454/bffaf519-f3d0-48ac-8927-57e4eb09e1b0)

**Comment by ramonhollands** [comment]
Ok, I will, thanks for your help!
